source/host/Option.cc

```diff
@@ -22,6 +22,10 @@ std::optional<EOption> CheckFlag(const char *arg) {
             return EOption::Suite;
         }
 
+        if (strcmp(verbose_arg, "ghost") == 0) {
```

Comment:
Any thoughts about making this an unordered_map lookup as well?

Reply:
I am very much not a fan of the dynamic allocation (and thus, non-constexpr nature) of `std::unordered_map`. However, I will probably use it just to make things easier to extend.

Reply:
After thinking about this carefully, a hashmap is not feasible here.

The map needs to be created and destroyed at runtime *every time* the function is called. We cannot make it static, due to heap problems with static initializers. `main.cc` is fine because we only ever call it once (although frankly, I'm considering the merit of not using hashmaps there either).